### ISSUE TITLE
work with apple wallet passbook

### ISSUE THREAD

**Issue by Mahmoud-shaban** [andrei-m-code/net-core-push-notifications#85]
can I use this library to send notification with apple wallet passbook ?
I don't have mobile app, only wallet passbook.

**Comment by PedroCavaleiro** [comment]
Yes it is possible to update the passbook, part of the process to update the passbook goes through sending a push notification, which is the objective of this library.

You will need to read the documentation further for it though

From the Apple Docs

> Create and send a push notification for each registered device. The notification uses the same certificate and private key that the creator of the pass used to sign the original, the push token registered by the device, and an empty JSON dictionary for the payload.

Full docs https://developer.apple.com/documentation/walletpasses/adding_a_web_service_to_update_passes#3737337

All other steps this library can't help, only with the push notification
